### PR TITLE
fix(common/models): Android API compat for upgraded correction-search

### PR DESCRIPTION
**Pull request by jahorton**
Legacy Android APIs are the new IE.

Took a few polyfills... and _polyfill editing_... to get everything patched up.

Screen Shot 2020-09-22 at 10 17 29 AM

### CODE REVIEW

**Comment by jahorton** [comment]
Realized this didn't need to be based off of the suggestion-refactor PR.  We'll probably want to merge this one earlier, actually.

File: common/predictive-text/polyfills/symbol-es6.min.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// From https://rousan.io/symbol-es6/, lightly modified to be worker-compatible for Android Lollipop.
```
Comment (darcywong00) [inline common/predictive-text/polyfills/symbol-es6.min.js:1]: What kind of modifications did you have to do to be "worker-compatible for Android Lollipop"?
in the event you need to re-create this.

Comment (jahorton) [reply common/predictive-text/polyfills/symbol-es6.min.js:1]: `typeof window?window:(self?self:global)`

There originally was no `self`-check; the polyfill assumed it was either in the `window` or that `global` would exist.  Naturally, being an old Android API, neither was true within the WebWorker.

Comment (jahorton) [reply common/predictive-text/polyfills/symbol-es6.min.js:1]: That was the only issue.

**Review by darcywong00 — APPROVED** [review]
lgtm. It worked in my emulator for Pixel 2 with API 21.

**Pull request merged**